src/vision/calib/MonoCalibrator.hpp

```diff
@@ -0,0 +1,62 @@
+#ifndef MONOCALIBRATOR_HPP
+#define MONOCALIBRATOR_HPP
+
+#include <ros/ros.h>
+#include "CalibrationSettings.hpp"
+#include <ctime>
```

Comment:
The majority of the calibration code was taken from the calib3d examples. I'll check through it to make sure that ctime is necessary.